packages/fxa-shared/nestjs/sentry/sentry.plugin.ts

```diff
@@ -28,9 +28,10 @@ import { Transaction } from '@sentry/types';
 import {
   ExtraContext,
   isApolloError,
+  isOriginallyHttpError,
   reportRequestException,
 } from './reporting';
-import { Inject } from '@nestjs/common';
+import { Inject, UnauthorizedException } from '@nestjs/common';
```

Comment:
@dschom You still have a couple of unused imports with `UnauthorizedException` here and in [reporting.ts](https://github.com/mozilla/fxa/pull/16217/files#diff-87f56dc70eb3ff04d28ea957acbaa51879d683ad98800d7e1a3139c3f2a70b1d) BTW